sdcm/report_templates/results_latency_during_ops_short.html

```diff
@@ -55,8 +55,12 @@ <h2>{{ operation }}</h2>
                         {% set lat_type_list = ['c-s P99'] %}
 
                         {% for lat_type in lat_type_list|sort %}
-                                {% if 'color' in results and lat_type in results['color'] and results['color'][lat_type] == 'red' %}
-                                    <tr style ="background-color: red">
+                                {% if 'color' in results and lat_type in results['color'] %}
```

Comment:
can we always provide some color? E.g. white when everything is fine?
Then we could drop all of this `if` statements related to color, just setting.

Reply:
we kind of always do, when the code that attributes the colors succeed (but we still wanted to protect this HTML form and not fail, if the color is missing), but in the `style` line, we can't attribute a parameter from the code AFAIU, because it should have come from a `css` style configuration, and i don't want to add one